Sources/Intermodular/Helpers/LinkPresentation/LinkPresentationView.swift

```diff
@@ -181,6 +181,7 @@ struct _LinkPresentationView<Placeholder: View>: Identifiable, View {
         
         do {
             if let metadata = try cache.decache(LPLinkMetadata.self, forKey: url) {
```

Comment:
I think you may have forgotten to delete this line, CI/CD seems to be failing because of this.

Reply:
-.- most sloppy me. Fixed!